tests/Plugin_Context_Tests.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Tests for the Plugin_Context class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Plugin_Context;
+
+class Plugin_Context_Tests extends WP_UnitTestCase {
+	public function set_up() {
+		parent::set_up();
+
+		$this->plugin_context = new Plugin_Context( 'plugin-check/plugin-check.php' );
```

Comment:
Can we apply https://github.com/10up/plugin-check/pull/47#discussion_r1054873951 changes here also?